Gems/Multiplayer/Code/Source/Debug/MultiplayerConnectionViewportMessageSystemComponent.cpp

```diff
@@ -213,8 +215,10 @@ namespace Multiplayer
             const float center_screenposition_y = 0.5f * viewportSize.m_height;
 
             // Fade out the toast over time
+            const size_t wordCount = m_centerViewportDebugToastText.find(' ') + 1; // Word count estimated by counting the number of spaces and adding 1.
```

Comment:
nit: this feels like a util we could call rather than an essential part of the "algorithm". OnRenderTick is already overlong IMHO

Reply:
Good eye, I found opportunity to consolidate both the toast and persistent debug text with a new DrawCenterViewportMessage(). Thanks